index.html

```diff
@@ -0,0 +1,23 @@
+---
+layout: default
+title: Welcome
+permalink: /
+---
+
+<h2>Welcome</h2>
```

Comment:
Because you have declared the `title` variable in the frontmatter, you can do {{ title }}. However, it'd be best to put this in the head include IMO.